.github/workflows/visual-regression.yml

```
@@ -15,8 +14,12 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    # Not intended for forks.
-    if: github.repository == 'optuna/optuna'
+    if: |
+      ( github.repository == 'optuna/optuna' ) && # Not intended for forks.
+      (
+         contains(github.event.pull_request.labels.*.name, 'safe-to-run-visual-regression-tests') ||
+         ( github.event_name == 'workflow_dispatch' )
+      )
```

Do you think it is possible to use an early return pattern here?

```suggestion
    # Not intended for forks.
    if: github.repository == 'optuna/optuna'
    if:  |
      (
         contains(github.event.pull_request.labels.*.name, 'safe-to-run-visual-regression-tests') ||
         (github.event_name == 'workflow_dispatch')
      )
```